scripts/cleanup.sh

```diff
@@ -6,5 +6,6 @@ set -o pipefail
 rm .github/CODEOWNERS
 rm .github/workflows/create-release-pr.yml
 rm .github/workflows/publish-release.yml
+rm .github/workflows/publish-site.yml
```

Comment:
I'm not sure if this still makes sense. It will cause the `main.yml` workflow to be broken, probably.